### ISSUE TITLE
Exporting PDF with Code Blocks

### ISSUE THREAD

**Issue by ThePooE** [MacDownApp/macdown#673]
If we were to export this,

```c
#include <stdio.h>
void apple(
    return 'apple';
)
```

The code and the language label would appear but the background block wouldn't appear

**Comment by ThePooE** [comment]
<img width="619" alt="screen shot 2016-10-10 at 14 32 21" src="https://cloud.githubusercontent.com/assets/4032361/19226398/6e778720-8ef6-11e6-8059-bf7014102337.png">

**Comment by franklinyu** [comment]
Mine works:

    begin block
    
    ```c
    #include <stdio.h>
    void apple(
        return 'apple';
    )
    ```
    
    end block

gives [test-0.pdf](https://github.com/MacDownApp/macdown/files/558795/test-0.pdf).

**Comment by ThePooE** [comment]
Yeah I was trying to figure out what's wrong and I got to the point that if you use github3 css, the blocks woundn't render.

**Comment by ThePooE** [comment]
Here is the sample css
https://gist.github.com/ThePooE/38fe2bfb6320f4a65d20d46bf46ce435

**Comment by franklinyu** [comment]
Yes I can confirm this. Neither the code block nor the language label shows any background color. Syntax highlight works though.

But the CSS is too large to locate the bug; it will help greatly if you would like to produce a "minimal working CSS".

**Comment by ThePooE** [comment]
Alright, I've trimmed down the CSS for you.

https://gist.github.com/ThePooE/1b096a16e2456e90a3e89785cb64e616
